File: app/src/main/java/com/orangepenguin/boilerplate/screens/username/UsernameView.java
package com.orangepenguin.boilerplate.screens.username;

import com.orangepenguin.boilerplate.BaseViewInterface;
import com.orangepenguin.boilerplate.model.User;

public interface UsernameView extends BaseViewInterface {
    void startDetailsActivity(User user);
    void setUsername(String s);
    void checkRememberCheckbox();
    void setUsernameError(String error);
}

File: app/src/main/java/com/orangepenguin/boilerplate/screens/username/UsernameView.kt
package com.orangepenguin.boilerplate.screens.username

import com.orangepenguin.boilerplate.BaseViewInterface
import com.orangepenguin.boilerplate.model.User

interface UsernameView : BaseViewInterface {
    fun startDetailsActivity(user: User)
    fun setUsername(s: String)
    fun checkRememberCheckbox()
    fun setUsernameError(error: String)
}
